group_vars/all.yml

```diff
@@ -24,7 +24,17 @@ kube_service_addresses: 10.254.0.0/16
 # Flannel internal network (optional). When flannel is used, it will assign IP
 # addresses from this range to individual pods.
 # This network must be unused in your network infrastructure!
-kube_overlay_network: 4.0.0.0/16
```

Comment:
Is't it reasonable to have more common option here? For example if we add another optional networking project we can easily reuse same variable.